src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -2,23 +2,20 @@
 :page-slug: custom-domains
 :page-description: How to configure your Koji to use a custom domain.
 
-The domain name for your Koji is its web address.
-A domain name consists of a *root domain*, such as `+++withkoji.com+++`, and usually one or more subdomains.
-For example, in the domain name `+++withkoji.com/~johnsmith789/my-video-game+++`, `+++~johnsmith789+++` and `+++my-video-game+++` are subdomains.
-The Koji platform assigns a domain name to your Koji when you publish it.
-
-You might want to assign a custom domain name to your Koji.
-A custom domain name can be much easier to remember, and it usually just looks better.
-Custom domains are about where you want to serve the experience.
+When you publish a Koji, it's automatically assigned a permalink based on your user name and lives on the `+++withkoji.com+++` domain.
+Example: `+++withkoji.com/~johnsmith789/my-video-game+++`.
+You have the option of additionally serving your Koji at a custom domain.
+This can be helpful if you already own a domain that ties into your marketing strategy, or if you want to create a URL that is easier to identify or verbalize.
+The custom domain feature allows you to work with an existing domain or one of our alternative root domains.
```

Comment:
What does "existing domain" mean in this sentence? I'm wondering if we can remove this sentence or if it's calling out another option (other than the two presented in the page).

Reply:
It's calling out the option of using a Koji root domain, which isn't mentioned prior to this sentence. How about:
"The custom domain feature allows you to work with a domain you own or one of our alternative root domains."

Reply:
Yes, I think it works without the sentence. Let's try it, and we can always change it if we get questions. Thanks!